vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties

```diff
@@ -19,5 +19,6 @@ transformer.from-headless-crawling.respect-no-follow=false
 transformer.from-headless-crawling.respect-no-index=false
 
 resource-checker.uri-to-ignore-regex=
+resource-checker.attributes-to-check=href,src
```

Comment:
do we really need this property? isn't it going to be always either href or src?

Reply:
> do we really need this property? isn't it going to be always either href or src?

1. Sites can have their own attributes to specify, for example, image source 
2. We may want to check only links (href) but not page resources (src)

Reply:
yes, nope

Reply:
but for a its always gonna be href

Reply:
some platforms don't follow specifications

Reply:
please provide markup example